README.rst

```diff
@@ -924,6 +924,13 @@ Get device group detail
 
     GET /api/v1/controller/group/{id}/
 
+Get device group from organization slug and common name of VpnClient's Certificate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/controller/{organization_slug}/certificate/{common_name}/group/
```

Comment:
[didn't we say to not bind it to a specific organization](https://github.com/openwisp/openwisp-controller/issues/491#issuecomment-860270178)?

It won't work for the use case we have at hand because multiple organization can use the same captive portal zone.

I think this can be:

`/api/v1/controller/cert/{common_name}/group/?organizations=slug1,slug2,slug3`

Reply:
We later decided that since the common names will be universally unique, it won't be required to have multiple organizations here.

I only need to change the URL scheme, the rest works as expected.

Reply:
seems you forgot to update the docs.

Let's also name this "Get device group from certificate common name".
Let's also mention the organization slugs in the example.
It would be good to provide a short explanation of how this is used, eg:

This endpoint can be used to retrieve group information and metadata by the common name of a certificate used in a VPN client tunnel, this endpoint is used in layer 2 tunneling solutions for firewall/captive portals.